http/label_service.go

```diff
@@ -334,23 +359,23 @@ func newGetLabelsHandler(b *LabelBackend) http.HandlerFunc {
 	}
 }
 
-type getLabelsRequest struct {
-	filter platform.LabelMappingFilter
+type getLabelsMappingRequest struct {
+	filter influxdb.LabelMappingFilter
 }
 
-func decodeGetLabelsRequest(ctx context.Context, r *http.Request, rt platform.ResourceType) (*getLabelsRequest, error) {
-	req := &getLabelsRequest{}
+func decodeGetLabelMappingsRequest(ctx context.Context, r *http.Request, rt influxdb.ResourceType) (*getLabelsMappingRequest, error) {
```

Comment:
Should this be decode GetLabelsMappingRequest ?

Reply:
Other way around; I'm changing the struct name to match.